typescript/dall-e-image-generation/index.ts

```diff
@@ -36,21 +36,13 @@ const generateImage = async (prompt: string) => {
       await download(openAiUrl, localPath);
 
       // Upload the image to Literal AI
-      const { objectKey, url: literalAiUrl } =
-        await literalClient.api.uploadFile({
-          path: localPath,
-          mime: "image/png",
-          threadId: literalClient.getCurrentThread().id,
-        });
-
-      const attachment = new Attachment({
+      const { url: literalAiUrl } = await literalClient.api.createAttachment({
         name: "result",
-        objectKey,
+        path: localPath,
         mime: "image/png",
+        threadId: literalClient.getCurrentThread().id,
```

Comment:
Since we're context-aware, I'd remove `threadId`.

Reply:
Actually, we only create a run here, I don't think it makes sense to pass a thread.

Reply:
done